ocs_ci/ocs/elasticsearch.py

```diff
@@ -0,0 +1,215 @@
+"""
+Deploying en Elasticsearch server for collecting logs from ripsaw benchmarks
+
+"""
+import os
+import logging
+import tempfile
+import urllib
+import urllib.error
+import base64
+import signal
+import subprocess
+import time
```

Comment:
plz separate imports by one line (built-in, 3rd party, local)

Reply:
done